Dockerfile

```
@@ -1,3 +1,9 @@
-FROM registry.gitlab.com/pages/hugo/hugo_extended:0.101.0
-CMD [ "serve", "-D", "--bind", "0.0.0.0" ]
-ENTRYPOINT [ "hugo" ]
+FROM registry.gitlab.com/pages/hugo/hugo_extended:0.101.0 as builder
+RUN mkdir -p /site
+WORKDIR /site
```

These can actually be collapsed down to just the `WORKDIR` directive according to the [docs](https://docs.docker.com/engine/reference/builder/#workdir) (it'll auto create the folder if it doesn't exist).

```suggestion
WORKDIR /site
```